### PR TITLE
Scroll lag fix

### PR DESCRIPTION
**Pull request by EmmanuelObua**
# Description

- Added stylings to section element which has SVG graphics, this styling aids at optimizing scrolling of the heaving graphics especially for phones which are mostly affected due to small GPU.
- The styling aids as welling it caching small graphics on first-time loads for faster scrolling hence reducing lags

### CODE REVIEW

File: assets/css/responsive.css [hunk 1]
```diff
@@ -166,8 +166,8 @@
 	}
 
 	#features .yellow-phone img {
-		top: 37%;
-		left: -45%;
+		top: 40%;
+		left: -62%;
```
Comment (mentumtech) [inline assets/css/responsive.css:170]: Why was this change needed/related for the scroll fix?

**Review by mentumtech — COMMENTED** [review]
Thanks @EmmanuelObua looks good, one question.

File: assets/css/responsive.css (hunk 1 above)
Comment (EmmanuelObua) [reply assets/css/responsive.css:170]: # Narration
Adding **content-visibility** CSS property on the section dom element was overlapping the cards on the margins on mobile, so I had to add some paddings on the left and right of the section element and then recenter the SVG of the yellow-phone by modifying the top and left elements as commited.
